ktor-server/ktor-server-cio/common/src/io/ktor/server/cio/CIOApplicationEngine.kt

```diff
@@ -108,20 +108,33 @@ public class CIOApplicationEngine(
         stopSuspend(gracePeriodMillis, timeoutMillis)
     }
 
-    private fun CoroutineScope.startConnector(host: String, port: Int): HttpServer {
-        val settings = HttpServerSettings(
-            host = host,
-            port = port,
-            connectionIdleTimeoutSeconds = configuration.connectionIdleTimeoutSeconds.toLong(),
-            reuseAddress = configuration.reuseAddress
-        )
+    private fun CoroutineScope.startConnector(connectorSpec: EngineConnectorConfig): HttpServer {
+        return when (connectorSpec) {
+            is UnixSocketConnectorConfig -> {
+                val settings = UnixSocketServerSettings(
+                    socketPath = connectorSpec.socketPath
+                )
```

Comment:
Should we pass `connectionIdleTimeoutSeconds` configuration here?
```suggestion
                val settings = UnixSocketServerSettings(
                    socketPath = connectorSpec.socketPath,
                    connectionIdleTimeoutSeconds = configuration.connectionIdleTimeoutSeconds.toLong(),
                )
```

Reply:
let me check